### ISSUE TITLE
Write Discussion Section (1 page)

### ISSUE THREAD

**Issue by twitte01** [twitte01/288R_Capstone#29]
- Give details about what (hyper)parameters you chose (e.g. why did you use X learning rate for gradient descent, what was your mini-batch size and why) and how you chose them. Did you do cross-validation, if so, how many folds? Before listing results, list and explain what your primary metrics are: accuracy, precision.
- Provide equations for the metrics if necessary. For results, have a mixture of tables and plots. For a classification problem, include a confusion matrix. Include performance metrics such as precision, recall, and accuracy. For regression problems, state the average error. Include visualizations of results, heatmaps, examples of where your algorithm failed and a discussion of why certain algorithms fail/succeed. Make sure to discuss the figures/tables in your main text throughout this section. Plots should include legends, axis labels, and have font sizes that are legible when printed.
